rust-runtime/aws-smithy-http-server/src/rejection.rs

```diff
@@ -185,6 +185,8 @@ pub enum RequestRejection {
     // error, but it would be a lot of effort for comparatively low benefit.
     /// Used when consuming the input struct builder.
     Build(crate::Error),
+
+    EnumVariantNotFound(Box<dyn std::error::Error + Send + Sync>),
```

Comment:
Add documentation. Mention this is the only one that does not take in `crate::Error` because the inner type is code-generated, unlike the other ones.